e2e/mhc_e2e_test.go

```diff
@@ -155,7 +154,7 @@ var _ = Describe("e2e - MHC", Label("MHC", labelOcpOnlyValue), func() {
 				Eventually(func(g Gomega) {
 					err := k8sClient.Get(context.Background(), ctrl.ObjectKey{Name: leaseName, Namespace: leaseNs}, lease)
 					g.Expect(k8serrors.IsNotFound(err)).To(BeTrue())
-				}, "1m", "5s").Should(Succeed(), "lease not deleted")
+				}, "5m", "5s").Should(Succeed(), "lease not deleted")
```

Comment:
this should not be needed, the lease should be deleted immediately after CR deletion...